File: PayMeBankEnd/src/main/java/com/example/PayMe/Entity/BankCard.java
package com.example.PayMe.Entity;

import com.fasterxml.jackson.annotation.JsonBackReference;
import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Data
public class BankCard implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(nullable = false,unique = true)
    private String number;
    private String ccv;
//    @Column(nullable = false)
//    private String pin;
//    @Column(nullable = false)
    private String cardType;
//    private Double amount;
//    @Column(nullable = false)
//    private String exp;
    @Column(nullable = false)
    private  String name;
    @JsonBackReference
    @ManyToOne(fetch = FetchType.LAZY,cascade = CascadeType.ALL)
    @JoinColumn(name = "user_id")
    private UserEntity user;

}

File: PayMe watch/app/src/main/java/com/se3/payme/models/BankCard.kt
package com.se3.payme.models

data class BankCard(
    val id:Long?=null,
    val number:String?=null,
    val ccv:String?=null,
//    val pin:String?=null,
    val cardType:String?=null,
//    val amount:Double?=null,
//    val exp:String?=null,
    val name:String?=null,
)